docker/etna-base/entrypoints/build.sh

```diff
@@ -32,11 +32,11 @@ if [ -z "$SKIP_RUBY_SETUP" ]; then
   if [ -e ../etna ] && grep 'monoetna.git' /app/Gemfile 1>/dev/null; then
     mkdir -p /tmp/
     rm -rf /tmp/.git
-    mv /app/vendor/bundle/*/ruby/*/bundler/gems/monoetna-*/.git /tmp/.git
+    cp -rf /app/vendor/bundle/*/ruby/*/bundler/gems/monoetna-*/.git /tmp/.git
```

Comment:
Sneaking this in -- I was having issues locally with `mv` failing for obscure reasons, and using `cp` seems more stable. This is supposed to go away once Zach has a chance to clean up Docker stuff...

Reply:
Sure, if this helps your build for now.  Yeah, I'm hoping to kill all of this off soon!